pkg/meta/model/table.go

```diff
@@ -50,6 +50,12 @@ const ExtraRowChecksumID = -4
 // ExtraCommitTSID is the column ID of column which holds the commit timestamp.
 const ExtraCommitTSID = -5
 
+// ExtraOriginTSID is the column ID of column which holds the origin timestamp.
+const ExtraOriginTSID = -6
```

Comment:
It's better to alloc a new ID llike normal columns instead of a preserved one if the column is store in the value